### PR TITLE
media-video/obs-studio: Version bump to 0.15.1.

### PR DESCRIPTION
**Pull request by Chiitoo**
Add ebuild for the [0.15.1 release] (https://github.com/jp9000/obs-studio/releases/tag/0.15.1).

Also, following a suggestion by 'username234' at #gentoo-proxy-maint, change depending directly on 'media-libs/x264' to 'media-video/ffmpeg:=[x264]'.

While it's possible to run the application without 'libx264', it does seem to be required at build time, with no option to disable it (it might be the only things used for the live-streaming part, after all).

Tested building 'media-video/ffmpeg' without x264, and then having 'media-libs/x264' installed, but the options to save local recordings using 'libx264' did not show up, so 'media-video/ffmpeg:=[x264]' seems to be required although their documentation made me think otherwise (I may also have missed something in the process of testing).

Since this is used for video recording, there are quite a bit of supported formats for containers and codecs, so there certainly might be quite a bit of dependency mapping to be done yet...

THE.  REST.  for you, @monsieurp!

### CODE REVIEW

File: media-video/obs-studio/obs-studio-0.15.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	pulseaudio? ( media-sound/pulseaudio )
+	truetype? (
+		media-libs/fontconfig
+		media-libs/freetype
+	)
+	v4l? ( media-libs/libv4l )
+"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
```
Comment (monsieurp) [inline media-video/obs-studio/obs-studio-0.15.1.ebuild]: Pick your choice: either `default` or `eapply_user` must be called during the `src_prepare()` phase.

**Comment by Chiitoo** [comment]
Done added 'default' to that which somehow was working without it before.

**Comment by monsieurp** [comment]
@mgorny uh? gentoo-mirror?

**Comment by mgorny** [comment]
I'd guess the commit landed in gentoo-mirror/gentoo before gentoo/gentoo. Which would suggest that the push to the latter failed somehow.

**Pull request closed without merging**